### PR TITLE
830 filter articles tags

### PR DESCRIPTION
**Pull request by Rowansdabomb**
Added a tag filter field to articles

### CODE REVIEW

File: dispatch/api/views.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ class ArticleViewSet(DispatchModelViewSet, DispatchPublishableMixin):
     model = Article
     serializer_class = ArticleSerializer
     lookup_field = 'parent_id'
+    update_fields = ('tags')
```
Comment (psiemens) [inline dispatch/api/views.py]: I don't think this is required

File: dispatch/api/views.py [hunk 2]
```diff
@@ -76,21 +77,28 @@ def get_queryset(self):
...
             queryset = queryset.filter(section_id=section)
+        
+        if tags is not None:
+            for tag in tags:
+                queryset = queryset.filter(tags__id=tag)
 
         if author is not None:
             queryset = queryset.filter(authors__person_id=author)
 
+        print(self.request.build_absolute_uri())
```
Comment (psiemens) [inline dispatch/api/views.py]: Remove print statement

File: dispatch/api/views.py [hunk 3]
```diff
@@ -222,8 +230,11 @@ def get_queryset(self):
         tags = self.request.query_params.getlist('tags', None)
         q = self.request.query_params.get('q', None)
 
+        print(self.request.query_params)
```
Comment (psiemens) [inline dispatch/api/views.py]: Remove print statement

File: dispatch/static/manager/src/js/pages/Articles/ArticleIndexPage.js [hunk 4]
```diff
@@ -2,10 +2,11 @@ import React from 'react'
 import { connect } from 'react-redux'
 
 import ItemIndexPage from '../ItemIndexPage'
-import SectionFilterInput  from '../../components/inputs/filters/SectionFilterInput'
-import AuthorFilterInput from '../../components/inputs/filters/AuthorFilterInput'
+import {AuthorFilterInput, SectionFilterInput, TagsFilterInput}  from '../../components/inputs/filters/'
+// import AuthorFilterInput from '../../components/inputs/filters/AuthorFilterInput'
 import articlesActions from '../../actions/ArticlesActions'
 import { humanizeDatetime } from '../../util/helpers'
+// import TagsFilterInput from '../../components/inputs/filters/TagsFilterInput'
```
Comment (psiemens) [inline dispatch/static/manager/src/js/pages/Articles/ArticleIndexPage.js]: Remove these comments

**Review by psiemens — CHANGES_REQUESTED** [review]

**Review by JamieRL — APPROVED** [review]
Other than what Peter has already mentioned, looks good

**Pull request merged**